README.md

```diff
@@ -10,6 +10,9 @@ END_METADATA -->
 This repository contains various resources for Vipps developers.
 We recommend starting with [Getting started](vipps-getting-started.md).
 
```

Comment:
Cool! 
Well -- it's kind of a hack, but if we don't want to show this in docusaurus, we can move it after that <!-- START_TOC --> flag.